### ISSUE TITLE
Long delay before bot connects to Discord

### ISSUE THREAD

**Issue by mpfaff** [kordlib/kord#452]
I frequently need to wait ~5 minutes for the bot to come Online in any guild. Occasionally, I can leave it for 8+ hours without any change.

**Comment by ByteAlex** [comment]
Hi!

I think you need to provide more information.

* How many servers is your bot in?
* How many shards did you log in?
* What version of Kord are you using?
* Code samples for where your bot gets stuck?

**Comment by mpfaff** [comment]
Sorry about that. Hope this clears things up:

* How many servers is your bot in? 1
* How many shards did you log in? Tried with default and number of cores. Same issue.
* What version of Kord are you using? 0.8.0-M7
* Code samples for where your bot gets stuck?

I have been unable to reproduce it at all since opening this issue, but I'll reply with a minimal sample once I can verify that the issue occurs with it.

I would like to note that this inconsistency has happened before. I've noticed the issue for nearly a month now and sometimes it just works fine for a few hours.
